autoload/fzf/OmniSharp.vim

```diff
@@ -83,3 +83,16 @@ let &cpoptions = s:save_cpo
 unlet s:save_cpo
 
 " vim:et:sw=2:sts=2
+
```

Comment:
All of the following needs to be moved up. `unlet s:save_cpo` and the modeline must be at the bottom of the file.